README.md

```diff
@@ -243,7 +243,7 @@ The standard Pi-hole customization abilities apply to this docker, but with dock
 
 Do not attempt to upgrade (`pihole -up`) or reconfigure (`pihole -r`).
 
-New images will be released for upgrades, upgrading by replacing your old container with a fresh upgraded image is the 'docker way'. Long-living docker containers are not the docker way since they aim to be portable and reproducible, why not re-create them often!  Just to prove you can.
+New images will be released for upgrades, upgrading by replacing your old container with a fresh upgraded image is the 'docker way'. Long-living docker containers are not the docker way since they aim to be portable and reproducible, why not re-create them often! **Just to prove you can.**
```

Comment:
Why did you change the styling here?

Reply:
```suggestion
New images will be released for upgrades, upgrading by replacing your old container with a fresh upgraded image is the 'docker way'. Long-living docker containers are not the docker way since they aim to be portable and reproducible, why not re-create them often!
```

I think this could even be removed.